tools/project_manager/show_all_good_word_suggestions.php

```diff
@@ -79,7 +99,9 @@
     // get all projects for this PM
     $projects = _get_projects_for_pm($pm);
 
+    $rejectAlllabel = _("Reject all suggestions");
     $submitLabel = _("Add selected words to Good Words List");
+    $rejectLabel = _("Add selected words to Bad Words List");
```

Comment:
Consider changing this to `$submitBadLabel` instead, since putting them on the bad list isn't rejecting them.